src/image/Image.js

```diff
@@ -62,13 +63,14 @@ class Image extends React.Component {
           testID="RNE__Image"
           {...attributes}
           onLoad={this.onLoad}
-          style={[
+          style={StyleSheet.flatten([
             StyleSheet.absoluteFill,
             {
-              width: style.width,
-              height: style.height,
+              width: width,
+              height: height,
             },
-          ]}
+            styleProps,
+          ])}
```

Comment:
@flyingcircle was just looking at this PR you merged. Any reason you went this route?

Could've simply done:

```jsx
style={StyleSheet.flatten([
  StyleSheet.absoluteFill,
  style
])}
```

Reply:
I did merge this, though I'm not the author. It didn't look too different from the previous commit, so I guess I didn't consider how to simplify it from even its previous version. I can say that I am now read up on the Stylesheet documentation ;P.